pyrene/src/components/Button/Button.tsx

```diff
@@ -50,7 +56,8 @@ const Button: React.FC<ButtonProps> = ({
 }: ButtonProps) => (
   <div styleName="buttonContainer">
     <button
-      type="submit"
+      // eslint-disable-next-line react/button-has-type
```

Comment:
Why is this needed? Does the rule not detect the default value?

Reply:
No, sonar complained about this.